File: fixtures/scale/objc/large/Families/Trithemius/TrithemiusVectors.m
#import "TrithemiusVectors.h"
#import "CLTestVector.h"

@implementation TrithemiusVectors

+ (NSArray *)all {
    NSMutableArray *vectors = [NSMutableArray array];
        [vectors addObject:[CLTestVector vectorWithPlaintext:@"ATTACKATDAWN" expected:@"EADNSDWSFFEY"]];
        [vectors addObject:[CLTestVector vectorWithPlaintext:@"THEQUICKBROWNFOX" expected:@"XOODKBYJDWWHBWIU"]];
        [vectors addObject:[CLTestVector vectorWithPlaintext:@"DEFENDTHEEASTWALL" expected:@"HLPRDWPGGJIDHNUIL"]];
    return vectors;
}

@end

File: fixtures/scale/swift/large/Families/Trithemius/TrithemiusVectors.swift
// Known-answer test data captured from TrithemiusCipher's own encode().
enum TrithemiusVectors {
    static func all() -> [TestVector] {
        return [
            TestVector(plaintext: "ATTACKATDAWN", expected: "EADNSDWSFFEY"),
            TestVector(plaintext: "THEQUICKBROWNFOX", expected: "XOODKBYJDWWHBWIU"),
            TestVector(plaintext: "DEFENDTHEEASTWALL", expected: "HLPRDWPGGJIDHNUIL"),
        ]
    }
}
